kstreams-app/build.gradle

```diff
@@ -14,15 +14,14 @@ plugins {
 }
 
 group = 'io.confluent'
-version = '0.0.9'
+version = '0.0.10'
 sourceCompatibility = "1.8"
 targetCompatibility = "1.8"
 
 def props = new Properties()
 file("../env_files/config.env").withInputStream { props.load(it) }
 
 ext {
-  kafkaVersion = '2.6.0'
   cpVersion = props.getProperty("CONFLUENT") ?: '6.0.0'
```

Comment:
@rspurgeon not required (because `CONFLUENT` is defined elsewhere) but WDYT about changing this to:

```
cpVersion = props.getProperty("CONFLUENT") ?: '6.2.0'
```

Reply:
Added in https://github.com/confluentinc/cp-demo/pull/377/commits/6b7bbd07ca8da7abd7d5b971d2c3f029591bf69f